suite-native/atoms/src/TitleHeader.tsx

```diff
@@ -0,0 +1,60 @@
+import { ReactNode } from 'react';
+
+import { TypographyStyle } from '@trezor/theme';
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+
+import { Text } from './Text';
+import { VStack } from './Stack';
+
+type TitleHeaderProps = {
+    title?: ReactNode;
+    titleVariant?: TypographyStyle;
+    subtitle?: ReactNode;
+    textAlign?: 'left' | 'center';
+};
+
+const textStyle = prepareNativeStyle<{ textAlign: TitleHeaderProps['textAlign'] }>(
+    (_, { textAlign }) => ({
+        textAlign,
+        width: '100%',
+    }),
+);
+
+export const TitleHeader = ({
+    title,
+    subtitle,
+    titleVariant = 'titleSmall',
+    textAlign = 'left',
+}: TitleHeaderProps) => {
+    const { applyStyle } = useNativeStyles();
+
+    return (
+        <VStack alignItems="center">
+            {title && (
+                <Text variant={titleVariant} style={applyStyle(textStyle, { textAlign })}>
+                    {title}
+                </Text>
+            )}
+            {subtitle && (
+                <Text color="textSubdued" style={applyStyle(textStyle, { textAlign })}>
+                    {subtitle}
+                </Text>
+            )}
+        </VStack>
+    );
+};
+
+export const CenteredTitleHeader = ({
```

Comment:
Not sure if I like that this file exports two components. Why did you decided to remove `CenteredTitleHeader` file? 

I suggest keeping two separate files for better overview which components do we have defined and also for consistency with other files.

Reply:
a16ef9a592